pilot/pkg/model/policyattachment.go

```diff
@@ -25,6 +25,7 @@ import (
 
 type policyTargetGetter interface {
 	GetTargetRef() *v1beta1.PolicyTargetReference
```

Comment:
Why don't we just remove the old one and have the newer one only in the interface?